cinema/models.py

```diff
@@ -1,9 +1,26 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=50)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors", blank=True)
+    genres = models.ManyToManyField(Genre)
```

Comment:
The related name is incorrect, you would not get all movies from `Actor` by referring to `actors`. Replace `actors` with `movies` as a related name. You could add a `realated_name` to the `genres` field as well.